src/software/ai/hl/stp/stp.cpp

```diff
@@ -169,7 +169,7 @@ std::vector<std::unique_ptr<Intent>> STP::getIntents(const World& world)
 }
 
 void STP::assignRobotsToTactics(const World& world,
-                                std::vector<std::shared_ptr<Tactic>> tactics) const
+                                std::vector<std::shared_ptr<Tactic>> &tactics) const
```

Comment:
Good idea

Reply:
I modified by reference because in `stp_tactic_assignment_test.cpp`, it's easier to check the tactics. If we return by value, would we still be able to do that?

Reply:
yes you can just check the returned value

Reply:
Done.